tests/userTest.js

```diff
@@ -548,42 +604,19 @@ describe('Main: Currently testing user management,', function () {
     await getCurrentUserInformationInteraction(provider)
     await changeUserAttributeInteraction(
       provider,
-      ' an existent user, attribute is allowed',
+      'an existent user, attribute is allowed',
       config.testUser,
       'key=email&value=asd%40a.com',
       meta => {
         ocsMeta(meta, 'ok', 100)
       }
     )
-    await getUserInformationInteraction(
-      provider,
-      ' to get user attribute of an existent user',
-      config.testUser,
 
-      new XmlBuilder('1.0', '', 'ocs')
-        .build(ocs => {
-          ocs.appendElement('meta', '', (meta) => {
-            return ocsMeta(meta, 'ok', '100')
-          }).appendElement('data', '', data => {
-            data.appendElement('enabled', '', 'true')
-              .appendElement('quota', '', quota => {
-                quota.appendElement('definition', '', 'default')
-              })
-              .appendElement('email', '', 'asd@a.com')
-              .appendElement('displayname', '', 'test123')
-              .appendElement('two_fafctor_auth_enabled', '', 'false')
-          })
-        })
-    )
     return provider.executeTest(async () => {
       const oc = createOwncloud()
       await oc.login()
       return oc.users.setUserAttribute(config.testUser, 'email', 'asd@a.com').then(data => {
         expect(data).toEqual(true)
-        return oc.users.getUser(config.testUser)
-      }).then(user => {
-        expect(typeof (user)).toEqual('object')
-        expect(user.email).toEqual('asd@a.com')
```

Comment:
why did you delete this part?

Reply:
This part is getting user after some attribute is set. But this kind of chaining doesn't work in provider side testing (provider side testing is limited to single interaction). So, the test will fail in provider side and also the test is for testing `setUserAttributes`.